src/backend/distributed/commands/database.c

```diff
@@ -191,6 +191,34 @@ PreprocessGrantOnDatabaseStmt(Node *node, const char *queryString,
 }
 
 
+/*
+ * Checks if the provided ALTER DATABASE statement is a SET TABLESPACE statement.
+ *
+ * This function takes a Node pointer representing a AlterDatabaseStmt, and checks
+ * if it is a SET TABLESPACE statement, which is used to move a table to a new
+ * tablespace.
+ *
+ * Parameters:
+ *   stmt: A pointer to a Node representing AlterDatabaseStmt.
+ *
+ * Returns:
+ *   true if the statement is a SET TABLESPACE statement, false otherwise.
+ */
```

Comment:
We don't really need to be that much verbose in function comments (when the function is simple enough).

Also, we start function comments with the name of the function. We also start with a capital letter when naming a function.

```suggestion
/*
 * IsSetTablespaceStatement returns true if given AlterDatabaseStmt contains
 * a DefElem to alter tablespace of the target database.
 */
```